web/src/components/Flag.tsx

```diff
@@ -15,11 +15,7 @@ interface CountryFlagProps
   size?: number;
 }
 
-export function CountryFlag({
-  zoneId,
-  size = DEFAULT_FLAG_SIZE,
-  ...props
-}: CountryFlagProps) {
+export function CountryFlag({ zoneId, size = DEFAULT_FLAG_SIZE }: CountryFlagProps) {
```

Comment:
This component should also accept a className to avoid breaking things :)

Reply:
Good spotting 🧐